content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst

```diff
@@ -56,27 +56,46 @@ clicking :guilabel:`Tickets` on the team's card (or by going to the Projects das
    :align: center
    :alt: Record time spent on a ticket.
 
+.. tip::
+   The ticket's timesheets can be recorded before or after the sales order is made, the order
+   doesn't matter.
+
 Link the Helpdesk ticket to the SO
 ==================================
 
-To link the :abbr:`SO (Sales Order)` to the ticket, start on the ticket form, click
-:guilabel:`Edit`, and then, click the :guilabel:`Sales Order Item` field. Next, select the
-:abbr:`SO (Sales Order)` that was created earlier from the drop-down menu. Lastly, click
-:guilabel:`Save` to connect the ticket and the :abbr:`SO (Sales Order)`.
+Once a timesheet is recorded, the :guilabel:`Sales Order Item` field will appear on the ticket
+form. To link the :abbr:`SO (Sales Order)` to the ticket, start on the ticket form and click
+:guilabel:`Edit`. Next, select the :abbr:`SO (Sales Order)` that was created earlier from the
+drop-down menu. Odoo will automatically filter the sales orders to only show sales orders that are
+connected to the ticket's customer. Lastly, click :guilabel:`Save` to connect the ticket and the
+:abbr:`SO (Sales Order)`.
 
-.. image:: reinvoice_from_project/link-so-ticket.png
+.. image:: reinvoice_from_project/link-soi-to-ticket.png
    :align: center
    :alt: Link the SO item to the ticket.
 
+Modify billing rates
+====================
+
+If a timesheet entry is recorded, but the agent does not want to bill the client for that time, go
+to the :guilabel:`Timesheets` tab and toggle on the visibility of the :guilabel:`Sales Order Item`
+column. Then, make sure to leave the non-billable timesheet entry's :abbr:`SOI (Sales Order Item)`
```

Comment:
Clarification:
Is this a column or is it actually a field or a choice/option?

```suggestion
If a timesheet entry is recorded, but the agent does not want to bill the client for that time, go
to the :guilabel:`Timesheets` tab and toggle on the visibility of the :guilabel:`Sales Order Item`
option.
```

Reply:
Hm, I would say it's both 😅 

Like, what I mean by the Sales Order Item "column" is this: (Screenshot of SOI column toggle: https://mivu.nimbusweb.me/s/share/7221077/p2zv8genu35hff15vte7)

Then, on each line you add to the timesheet, you fill out the information like the *Date*, *Employee*, *Description*, and *Duration* for the entry. And the Sales Order Item can be left blank or you can select an option from the drop-down menu. (Screenshot of SOI drop-down: https://mivu.nimbusweb.me/s/share/7221100/3mg66qnbk8fuopnmqarh)

I changed the `:abbr:` to a `:guilabel:` on Line 82, and tried to clarify the wording. Let me know if it's still confusing though!